vllm/entrypoints/openai/api_server.py

```diff
@@ -619,6 +620,28 @@ async def fake_stream_generator() -> AsyncGenerator[str, None]:
     else:
         served_model = args.model
 
+    if args.chat_template is not None:
```

Comment:
can you move the try except into a function? thank you making it robust!

Reply:
Yeah, will do.

Reply:
This is done in the new PR